timesketch/lib/utils.py

```diff
@@ -53,11 +56,18 @@ def read_and_validate_csv(path):
         if missing_fields:
             raise RuntimeError(
                 u'Missing fields in CSV header: {0:s}'.format(missing_fields))
-
         for row in reader:
+            if u'timestamp' not in csv_header and u'datetime' in csv_header:
+                try:
+                    row['timestamp'] = str(int(time.mktime(
+                        parser.parse(row["datetime"]).timetuple())))
```

Comment:
We can make them microseconds by `*1000`, but `psort`'s output is still datetime-seconds: `1970-01-01T00:00:00+00:00`